Comp2022/src/main/cpp/subsystems/Drivetrain.cpp

```diff
@@ -658,34 +658,27 @@ void Drivetrain::MoveWithLimelightInit(bool m_endAtTarget)
         double ty = robotContainer->m_vision.GetVertOffsetDeg();
         m_limelightDistance = m_slope * ty + m_distOffset;
         m_targetDistance = m_limelightDistance;
+        spdlog::info("DTL Using Current Distance {}", m_targetDistance);

```

Comment:
what does current distance mean? as in the current distance that is set as the target, or the actual current position of the robot? maybe modify the print statement; it's a little confusing